src/commands/translation_status.rs

```diff
@@ -0,0 +1,81 @@
+pub async fn translation_status(config: &fpm::Config) -> fpm::Result<()> {
+    // it can be original package or translation
+    let translation_status = if let Some(ref original) = config.package.translation_of.as_ref() {
+        translation_package_status(config, original).await?
+    } else if !config.package.translations.is_empty() {
+        original_package_status().await?
+    } else {
+        return Err(fpm::Error::UsageError {
+            message: "neither translation_of nor translations is set".to_string(),
+        });
+    };
+
+    print_translation_status(&translation_status);
+    Ok(())
+}
+
+async fn translation_package_status(
+    config: &fpm::Config,
+    original: &fpm::Package,
+) -> fpm::Result<std::collections::BTreeMap<String, TranslationStatus>> {
+    let original_path = config.root.join(".packages").join(original.name.as_str());
+    let snapshots = fpm::snapshot::get_latest_snapshots(&original_path).await?;
```

Comment:
When I see this I wonder if `original_path` is needed by this function as well? This is an expensive operation for me as a PR reviewer. If not and you need `original_path` only for calculating snapshots, use a block. Also if you only need `original_path` insider `get_latest_spanshots()` then the logic for calculating it must belong inside `get_latest_spanshots()`. Pass `config` as much as possible and let everyone calculate what they want, `translation_package_status()` knowing internal logic of `get_latest_snapshots()` is just wrong.